### PR TITLE
Fix TypeError with new version of Pygments

### LINKED ISSUE #1935: TypeError when starting server
After installing follow the instructions here: https://docs.dmoj.ca/#/site/installation

I try to run the server with the command `python3 manage.py runserver 0.0.0.0:8000` and try to access it and get the following error. How to fix it?

image

### PR DESCRIPTION
**Pull request by Ninjaclasher**
Fixes #1935.

### CODE REVIEW

File: judge/highlight_code.py [hunk 1]
```diff
@@ -23,15 +16,13 @@ def _wrap_code(inner):
...
-        def wrap(self, source, outfile):
-            return self._wrap_div(self._wrap_pre(_wrap_code(source)))
-
     def highlight_code(code, language, cssclass='codehilite'):
         try:
             lexer = pygments.lexers.get_lexer_by_name(language)
         except pygments.util.ClassNotFound:
             return _make_pre_code(code)
 
         # return mark_safe(pygments.highlight(code, lexer, HtmlCodeFormatter(cssclass=cssclass, linenos='table')))
```
Comment (int-y1) [inline judge/highlight_code.py:25]: Delete this. I don't think DMOJ users are ready for `linenos='table'`.

File: judge/highlight_code.py [hunk 2]
```diff
@@ -7,13 +7,6 @@ def _make_pre_code(code):
...
-def _wrap_code(inner):
-    yield 0, '<code>'
-    for tup in inner:
-        yield tup
-    yield 0, '</code>'
-
-
 try:
     import pygments
     import pygments.lexers
```
Comment (int-y1) [inline judge/highlight_code.py:12]: (This comment is for line 13)
Change to `import pygments.formatters`. The `.html` is a historical artifact.

Comment (Ninjaclasher) [reply judge/highlight_code.py:12]: Done

File: judge/highlight_code.py (hunk 1 above)
Comment (Ninjaclasher) [reply judge/highlight_code.py]: Done

**Review by int-y1 — APPROVED** [review]
LGTM.
There are no issues with backwards compatibility. Even Pygments 2.4.0 worked fine.

**Review by quantum5 — APPROVED** [review]

**Pull request merged**